### PR TITLE
Feat: Added "Keyboard.None"

### LINKED ISSUE #10331: Add "Keyboard.None"
### Description

Currently I can only choose _what_ type of keyboard I want for a specific `Entry`, but in order to fully disable the keyboard I have to do some weird workarounds with `#if`s and custom renderers involved: https://stackoverflow.com/questions/73199602/need-a-way-to-hide-soft-keyboard-in-mauis-editor-entry-fields (the first link in google search for me). I think such capability should be implemented on platform layer and be available to all MAUI users without that much effort to use it.

The end result of this proposal should look like this (taken from the article about how to implement such feature in Xamarin, which also didn't have a build-in solution for it):

### Public API Changes

In https://github.com/dotnet/maui/blob/main/src/Core/src/Primitives/Keyboard.cs:
```diff
public class Keyboard
{
    ...
+    public static Keybord None
+    {
+        ...
+    }
}
```

https://github.com/dotnet/maui/blob/main/src/Core/src/Converters/KeyboardTypeConverter.cs should also be modified.

### Intended Use-Case

Well, see almost any calculator app for reference. It provides an input field but doesn't trigger the system keyboard when tapped/clicked. In general the feature would be _very_ useful when the app needs a specific control over what user is typing. Such apps usually have their own keyboard or a limited set of buttons for data input

### PR DESCRIPTION
**Pull request by GabrielGavrilov**
### Description of Change

Added "Keyboard.None"

### Issues Fixed

Fixes #10331

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/maui?pullRequest=10375) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/maui?pullRequest=10375) before we can accept your contribution.<br/><br/>:x: GabrielGavrilov [sign now](https://cla.dotnetfoundation.org/dotnet/maui?pullRequest=10375)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/maui?pullRequest=10375) it.</sub>

File: src/Core/src/Primitives/Keyboard.cs [hunk 1]
```diff
@@ -74,11 +76,17 @@ public static Keyboard Url
 		{
 			get { return s_url ??= new UrlKeyboard(); }
 		}
+		
+		/// <include file="../../docs/Microsoft.Maui/Keyboard.xml" path="//Member[@MemberName='None']/Docs/*" />
+		public static Keyboard None
+		{
+			get { return s_none ??= new NoneKeyboard(); }
```
Comment (jfversluis) [inline src/Core/src/Primitives/Keyboard.cs:83]: The type `NoneKeyboard` doesn't exist, does it?

File: src/Core/src/Primitives/Keyboard.cs [hunk 2]
```diff
@@ -74,11 +76,17 @@ public static Keyboard Url
 		{
 			get { return s_url ??= new UrlKeyboard(); }
 		}
+		
+		/// <include file="../../docs/Microsoft.Maui/Keyboard.xml" path="//Member[@MemberName='None']/Docs/*" />
```
Comment (jfversluis) [inline src/Core/src/Primitives/Keyboard.cs:80]: You'll also have to add the actual docs for it, not just replace the member name here

**Review by jfversluis — CHANGES_REQUESTED** [review]
I don't think this implementation is complete?

**Pull request closed without merging**